code/backend/pages/02_Explore_Data.py

```diff
@@ -40,21 +39,18 @@
 
 try:
     vector_store_helper: AzureSearchHelper = AzureSearchHelper()
-    search_client = vector_store_helper.get_vector_store().client
+    search_client = vector_store_helper.get_client()
     # get unique document names by getting facets for title field
     results = search_client.search("*", facets=["title"])
     unique_files = [filename["value"] for filename in results.get_facets()["title"]]
     filename = st.selectbox("Select your file:", unique_files)
     st.write("Showing chunks for:", filename)
 
     results = search_client.search(
-        "*", select="title, content, metadata", filter=f"title eq '{filename}'"
+        "*", select="title, content, chunk", filter=f"title eq '{filename}'"
     )
 
-    data = [
-        [json.loads(result["metadata"])["chunk"], result["content"]]
-        for result in results
-    ]
+    data = [[result["chunk"], result["content"]] for result in results]
```

Comment:
so that metadata was just encoded to we could load it here?!?!

Reply:
This is one of the usages, it's used in a couple of other places, not sure why it needs to. I _probably_ won't remove it when/if I do this for real to keep changes to a minimum, I was just being lazy in this POC avoiding json.dumping it.